### PR TITLE
Add BusinessSubscription, business.getSubscriptions(id)

### PR DESCRIPTION
**Pull request by chenhunghan**
For this endpoint `GET /businesses/:id/subscriptions` https://github.com/lensapp/lenscloud/issues/1938

### CODE REVIEW

**Comment by chenhunghan** [comment]
Tests fixed in https://github.com/lensapp/lens-platform-sdk/pull/334

File: src/BusinessService.ts [hunk 1]
```diff
@@ -61,6 +61,38 @@ export type Business = {
...
+  nextRenewalDate: string;
+  /**
+   * Total number of seats in this subscription, including unassigned and assigned. (Recurly subscription["quantity"])
+   */
+  seats: number;
+  /**
+   * The seats that have not been assigned to a user yet.
+   * = `seat` field - (number of subscription id in user_subscriptions table)
+   */
+  aviableSeats: number;
```
Comment (nevalla) [inline src/BusinessService.ts:93]:
```suggestion
  availableSeats: number;
```

**Review by nevalla — APPROVED** [review]
Found one typo, otherwise LGTM

**Comment by nevalla** [comment]
> Tests fixed in #334

Approved. Let's merge it first and rebase this branch to get this green as well.

File: src/BusinessService.ts (hunk 1 above)
Comment (chenhunghan) [reply src/BusinessService.ts]: Thanks, fixed.

**Pull request merged**